bryft/src/pages/AboutPage/AboutPage.js

```diff
@@ -0,0 +1,58 @@
+import "./AboutPage.css";
+import Navbar from "../../components/Navbar/Navbar";
+import IconButton from "@mui/material/IconButton";
+import Email from "@mui/icons-material/Email";
+import Instagram from "@mui/icons-material/Instagram";
+import LinkedIn from "@mui/icons-material/LinkedIn";
+import { Link } from 'react-router-dom';
+
+import alan from '/Users/sophialee/Desktop/for git/Bryft/bryft/src/components/alanheadshot.jpeg';
```

Comment:
Can we make these paths relative instead of absolute? Right now it only works on your own computer haha